packages/account/Scarb.toml

```diff
@@ -1,16 +1,23 @@
 [package]
 name = "openzeppelin_account"
+readme = "README.md"
+keywords = [
+    "openzeppelin",
+    "starknet",
+    "cairo",
+    "contracts",
+    "standards",
+    "account"
+]
```

Comment:
Maybe I'm wrong here, but I think you should be able to inherit keywords from the main workspace and add specific ones on top right? I think that should be the case if we keep `keywords.workspace = true`. That way we can have 2-3 main ones as part of the workspace package and then add specific ones(like `account` in this case) to individual crates.

Reply:
Yeah, I saw the TG chat. Thanks for looking into it anyways!